EnglishFilter/sections/antiadblock.txt

```diff
@@ -544,6 +544,9 @@ kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74014
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,xmlhttprequest,redirect=nooptext,domain=pinayrare.net
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=pinayrare.net
```

Comment:
It should be like the example below
```
! https://github.com/AdguardTeam/AdguardFilters/issues/74014
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=pinayrare.net
||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=pinayrare.net
```

Reply:
What's the difference @zloyden ?

Reply:
This is necessary for our compiler and we have this order of rules.
Also `$script,redirect=googlesyndication-adsbygoogle` is enough for redirect rule, but you've missed the `important` modificator for this one.

Reply:
Ok.